omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,59 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+        $scope.loadTags =function (personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
Note that am saying you need to call loadTags() inside init() AND NOT declaring it inside init()